### ISSUE TITLE
-r (re-scheduling stale-scheduled tasks)

### ISSUE THREAD

**Issue by linuxcaffe** [linuxcaffe/tw-sched-sh#10]
TL/DR; The -r re-schedule option comes in 3 parts; 
1) if any matching tasks are past their sched:date,. prompt comes up with option to ckear those sched:dates and add a +sch tag. 
2) any matching tasks with (configurable) +sch tag, are listed as candidates
3) once a new sched:date is applied, the +sch tag is removed.

This is used to re-schedule tasks that were not completed at their scheduled time or date, and to put any task into the to-be-scheduled cue. When the sched date is "stale" it should be corrected immediately, as it's no longer +READY, won't normally show up in the candidates list, and you should have done it yesterday! 

`Re-schedule N tasks with stale sched:dates (Y/n) ?`

if "Y" then stale dates are cleared with;

`task sched.before:now mod sched: +sch`

Then matching +sch-tagged tasks are listed as candidates (if not "Y" then without clearing stale ones first) 
Tasks must be +PENDING, but they can be +READY or not., as long as they have the tag., 
After a task is rescheduled, as a last step, the +sch tag is removed.
